code/datums/dna.dm

```diff
@@ -11,13 +11,15 @@
 	var/list/mutations = list()   //All mutations are from now on here
 	var/list/temporary_mutations = list() //Timers for temporary mutations
 	var/list/previous = list() //For temporary name/ui/ue/blood_type modifications
-	var/mob/living/carbon/holder
+	var/mob/living/holder

```

Comment:
"holder" should pretty much always be a carbon, but I fear brain mob (now a mob/living) doing something stupid (despite the fact that holder is currently null for the dna stored in brains, but you never know...